Chef/Cookbooks/Relativity/attributes/default.rb

```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']
 
 default['file']['installers']['default_destination_folder'] = 'C:\\Chef_Install'
+default['file']['installers']['devvm_installer_files_path'] = ENV['devvm_installer_files_path']

```

Comment:
Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

Reply:
Done